mobify/progressive-web-scaffold
# Update README to work with build-sprites

Update README to work with build-sprites

 **JIRA**: n/a
 **Linked PRs**: n/a

## Changes
- Update README to work with build-sprites

## How to test-drive this PR
- add, remove or edit svg icons in `app/static/svg` folder
- run `npm run build-sprites` to make sure that updated `sprite.svg` is working.
